src/vmshepherd/presets/preset.py

```diff
@@ -83,36 +83,41 @@ def _reset_counters(self):
         """
         if if_older_than is None or time.time() - self._vms_refresh_time > if_older_than:
             self._vms_refresh_time = time.time()
-            self._vms = await self.iaas.list_vms(self.name)
+            self._runtime_vms = await self.iaas.list_vms(self.name)
+        else:
+            runtime_data = await self.runtime_mgr.get_preset_data(self.name)
+            self._vms = runtime_data.iaas['vms']
 
     async def manage(self):
         """ Manage function docstring"""
-
         await self.refresh_vms()
```

Comment:
I don't see the usage of `if_older_than` parameter anywhere, does it mean that the vm list will be refreshed from IaaS provider every time? I thought the whole point of this change ([together with postgres-runtime-driver PR](https://github.com/DreamLab/vmshepherd-runtime-postgres-driver/pull/2/files)) was to cache vms data in runtime

Reply:
You are right, the logic was changed.